### ISSUE TITLE
Hashing fails, prevents updating system images

### ISSUE THREAD

**Issue by kylechase** [jack-broadway/vyos-modular#9]
Building an image with the following fails

```
python3 main.py build_iso -b current -r current
cp build/baseiso.iso dist/isos
python3 main.py build -c config.yml
```
The  md5sum.txt file doesn't seem to get updated after the tailscale module is applied.

I checked manually with md5sum -c md5sum.txt, and I will attach the output file here.

I will need to look into this a little more, it might be an upstream issue.

[md5sum-fail.txt](https://github.com/jack-broadway/vyos-modular/files/13835661/md5sum-fail.txt)
[config.yml.txt](https://github.com/jack-broadway/vyos-modular/files/13835670/config.yml.txt)

**Comment by kylechase** [comment]
Just confirming that the same results occur when using sample-config.yml

**Comment by jack-broadway** [comment]
No it is an issue with vyos-modular, I admittedly haven't done an in place upgrade before on my infra as I tend to run in HA pairs. This should be an easy enough fix :)

**Comment by jack-broadway** [comment]
@kylechase Problem should be fixed in the 2.1.0 release
